src/ExtensionsSample/ErrorNotifier.cs

```diff
@@ -21,18 +18,9 @@ public class ErrorNotifier
         private const string NotificationUriSettingName = "AzureWebJobsErrorNotificationUri";
         private readonly string _webNotificationUri;
         private readonly HttpClient _httpClient = new HttpClient();
-        private readonly SendGridAPIClient _sendGrid;
```

Comment:
That's a good idea. 

---
In reply to: [132522405](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/261#discussion_r132522405) [](ancestors = 132522405)